src/Gateway/Validator/Constraints/IsCanSavePaymentMethodValidator.php

```diff
@@ -19,6 +21,8 @@ final class IsCanSavePaymentMethodValidator extends ConstraintValidator
 {
     private PayPlugApiClientFactory $apiClientFactory;
 
+    private const GATEWAYS_SKIP = [PayPlugGatewayFactory::FACTORY_NAME, OneyGatewayFactory::FACTORY_NAME];
```

Comment:
Because we dont need to check in this validator if the key is live or not and we dont need to check if payment method  is enabled true for payplug gateway and oney gateway

Reply:
Thanks !